app/views/templates/_news_and_documents.html.erb

```diff
@@ -26,11 +26,15 @@
         if articles.any? %>
           <articles>
 
-            <% articles.each do |a| %>
```

Comment:
As discussed, instance variables will be set in the view if there is no obvious controller to be used for the comfy templates.